plugins/UM3NetworkPrinting/NetworkClusterPrinterOutputDevice.py

```diff
@@ -243,6 +243,11 @@ def requestWrite(self, nodes, file_name=None, filter_by_machine=False, file_hand
         self._selected_printer = self._automatic_printer  # reset to default option
         self._request_job = [nodes, file_name, filter_by_machine, file_handler, kwargs]
 
+        # the build plates to be sent
```

Comment:
I think all changes to this file will conflict heavily with @nallath's rewrite of the output devices. It might have to be re-implemented in that branch.